DualityPlugins/Tilemaps/Tilemaps/Components/ICmpTilemapRenderer.cs

```diff
@@ -47,6 +47,13 @@ public interface ICmpTilemapRenderer : ICmpRenderer
 		/// <returns></returns>
 		Point2 GetTileAtLocalPos(Vector2 localPos, TilePickMode pickMode);
 		/// <summary>
+		/// Gets the local position of the specified tile. The function does not check if the point is a valid tile position.
+		/// </summary>
+		/// <param name="tile">The position of the tile.</param>
+		/// <param name="alignment">Determines the point of the tile that is returned.</param>
+		/// <returns></returns>
+		Vector2 GetLocalPosAtTile(Point2 tile, Alignment alignment = Alignment.Center);
```

Comment:
Okay, if I'm getting it right, if there was an implementation for, e.g a hexagonal tilemap renderer, that alignment wouldn't have a lot of sense.